### DISCUSSION TITLE
restore ability to click to previous reblog

### DISCUSSION THREAD

**Discussion by elsewyse** [AprilSylph/XKit-Rewritten#1092] [Suggestion Box]
Now that tumblr has changed what clicking on a blog title does (having it go to the blog, not the reblogged post), it's not easy to get to the previously-reblogged post. Any chance we could see a fix for this?

Thank you for all your hard work! <3

**Comment by marcustyphoon** [comment]
This has been added to Tweaks in v0.20.9 (#1084), which is out for Chrome and should be out for Firefox shortly once it passes through Mozilla's review process!

**Reply by elsewyse** [reply]
!! You're a wonder and a delight-- thank you so much!
